config.yml

```diff
@@ -2,10 +2,10 @@ version: 1.0
 debug: true
 network:
   type: tcp
-  address: ":5001"
+  address: ":5004"
 settings:
-  mode: parallel
+  mode: serial
```

Comment:
Has to be serial or the PLC will fall over and die. Not at all clear which part of the config makes this happen, but that is another question. This happens to work.